weekly-mission(week2)/src/constants/errorMessage.js

```diff
@@ -0,0 +1,18 @@
+const ERROR_MESSAGE = Object.freeze({
+  EMAIL: {
+    BLANK: '이메일을 입력해 주세요.',
+    INVALID: '올바른 이메일 주소가 아닙니다.',
+    INCORRECT: '이메일을 확인해 주세요.',
+    DUPLICATE: '이미 사용 중인 이메일입니다.',
```

Comment:
사소하지만 이렇게가 더 좋을 것 같아요!
```suggestion
    DUPLICATED: '이미 사용 중인 이메일입니다.',
```